### ISSUE TITLE
Cannot read property 'status' of undefined 

### ISSUE THREAD

**Issue by rattrayalex-stripe** [KnisterPeter/vscode-github#278]
I'm not sure what the issue is here. I am using Github Enterprise (GHE v11) which may be related.

I can run `Github: Browse current file` successfully, but the following commands error:

- Github: Browse project
- Github: Browse open issue...
- All others I could find, really

I can access this url in my browser successfully:
`https://<my ghe hostname>/api/v3/repos/<my org>/<my repo>?access_token=<my personal access token>` so I don't think it's an auth issue (?)

```
git config --local --get remote.origin.url
GET https://<my ghe hostname>/api/v3/repos/<my org>/<my repo>
Cannot read property 'status' of undefined
TypeError: Cannot read property 'status' of undefined
	at Object.<anonymous> (/Users/rattrayalex/.vscode/extensions/KnisterPeter.vscode-github-0.24.0/out/src/provider/github/api.js:51:37)
	at Generator.throw (<anonymous>)
	at rejected (/Users/rattrayalex/.vscode/extensions/KnisterPeter.vscode-github-0.24.0/out/src/provider/github/api.js:14:65)
	at <anonymous>
	at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by KnisterPeter** [comment]
I'll inspect this. Thanks for reporting.

**Comment by KnisterPeter** [comment]
Released as 0.24.1
